tests/contrib/djangorestframework/app/views.py

```diff
@@ -1,11 +1,17 @@
+import django
 from django.conf.urls import include
-from django.conf.urls import url
 from django.contrib.auth.models import User
 from rest_framework import routers
 from rest_framework import serializers
 from rest_framework import viewsets
 
 
+if django.VERSION < (2, 0, 0):
+    from django.conf.urls import url
+else:
+    from django.urls import re_path as url
```

Comment:
same as above, I'd prefer not to remap `re_path` to `url`

Reply:
I mapped it to "handler". The reason why I made this change was to keep this pr small. In hindsight I should've prioritized readability